src/System.Management.Automation/utils/ClrFacade.cs

```diff
@@ -60,7 +60,7 @@ internal static IEnumerable<Assembly> GetAssemblies(string namespaceQualifiedTyp
         {
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-macos`
Test-Connection.Ping.Force IPv4 with explicit PingOptions
```powershell
Cannot index into a null array.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 106
106:             $result1.Replies[0].Address              | Should -BeExactly $realAddress

```
Test-Connection.TraceRoute.Quiet works
```powershell
Expected $true, but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 258
258:             $result | Should -BeTrue
```